custom_components/tahoma/cover_devices/tahoma_cover.py

```diff
@@ -128,19 +125,6 @@ def is_closed(self):
 
         return None
 
-    @property
-    def icon(self):
```

Comment:
Replace By a sensor.

Reply:
@iMicknl About this one, I just checked, the state is returned only when the issue occured. Meaning we cannot create the sensor during the setup 99% of the time. What do you think about hardcoding it ?

Reply:
@tetienne hmm, this is a hard one. Should we create an issue for this one and tackle it in a follow up PR? 
I think that `CORE_PRIORITY_LOCK_TIMER_STATE` is always present, where `IO_PRIORITY_LOCK_ORIGINATOR_STATE` is only present when there is a reason (lock timer state > 0).

Reply:
Agreed. We can rollback this and focus on it later.

Reply:
Done